x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -217,8 +218,17 @@ export function WorkspacePanel({
           data: event.data,
         });
       }
+      if (isLensEditEvent(event)) {
+        if (activeVisualization?.onEditAction) {
```

Comment:
Could combine these two:

```suggestion
      if (isLensEditEvent(event) && activeVisualization?.onEditAction) {
```